global_body_planner/src/planning_utils.cpp

```diff
@@ -893,6 +944,7 @@ bool isValidState(const State &s, const PlannerConfig &planner_config,
   }
 
   // Find yaw, pitch, and their sines and cosines
+  // TODO(jcnorby): Add getRollFromState
```

Comment:
Does this need an issue?

Reply:
Same as before, we could but its low priority.